onnxruntime/core/providers/cpu/tensor/unique.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+
+namespace onnxruntime {
+class Unique final : public OpKernel {
+ public:
+  Unique(const OpKernelInfo& info) : OpKernel(info) {
```

Comment:
explicit ?

Reply:
Strictly speaking, sure. Practically speaking, not sure there are any scenarios where a Unique instance would get implicitly constructed from an OpKernelInfo reference.

Reply:
I think this `explicit` was being enforced in all the PRs (I know I got such a comment and started adding it). Maybe it is better to add for the sake of consistency.